Javascript/README.md

```diff
@@ -49,6 +49,23 @@ if (time < 60) {
 }
 ```
 
+There is an objective reason for this. If you write the following code
+
+```javascript
+function myFunction()
+{
+	return
+	{
+		property1: 'value1',
+		property2: 'value2'
+	};
+}
+```
+
+`return\n` will be interpreted as `return;\n` and your function will be broken. Thus, for consistency matter, we want
+any curly braces to be on the same line as whatever they open.
```

Comment:
But you can't have block brace after a `return`.
Here, this rule is about block brace.
The rule should not apply to object definition brace. For instance i could write this:
```javascript
function myFunction() {
	return [
		{ prop1: 1, prop2: 2 },
		{ prop1: 3, prop2: 4 }
	];
}
```
in which the curly braces don't open on the same line.